src/vs/workbench/services/extensions/electron-browser/localProcessExtensionHost.ts

```diff
@@ -149,10 +151,12 @@ export class LocalProcessExtensionHost implements IExtensionHost {
 		if (!this._messageProtocol) {
 			this._messageProtocol = Promise.all([
 				this._tryListenOnPipe(),
-				this._tryFindDebugPort()
+				this._tryFindDebugPort(),
+				this._telemetryService.getTelemetryInfo()
```

Comment:
I would certainly not bock the extension host startup to get the telemetry info. We do not seem to wait for the telemetry info in the renderer so I would not wait here unless needed?